build.assets/Dockerfile

```
@@ -10,7 +10,7 @@
 ## LIBFIDO2 ###################################################################
 
 # Build libfido2 separately for isolation, speed and flexibility.
-FROM buildpack-deps:20.04 AS libfido2
+FROM buildpack-deps:latest AS libfido2
```

Maybe:

```suggestion
FROM buildpack-deps:22.04 AS libfido2
```

as it is the most-recent LTS. Same for others.